google/cloud/bigtable/examples/stock_backtesting/backtesting_main.cc

```diff
@@ -0,0 +1,258 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START examples_backtesting_main]
+#include "google/cloud/bigtable/cell.h"
+#include "google/cloud/bigtable/filters.h"
+#include "google/cloud/bigtable/table.h"
+#include "google/cloud/internal/absl_str_cat_quiet.h"
+#include "google/cloud/status_or.h"
+#include "absl/strings/ascii.h"
+#include "absl/strings/numbers.h"
+#include "absl/strings/str_split.h"
+#include "absl/time/civil_time.h"
+#include "absl/time/time.h"
+#include "google/protobuf/text_format.h"
+#include "strategy.pb.h"
```

Comment:
The builds are breaking on this include. Maybe you want:
```
#include "google/cloud/bigtable/examples/strategy.pb.h"
```
or maybe that but with `stock_backtesting` ? Does it depend on the package defined in the proto?

I really don't know.... good luck.

Reply:
Made it build.